docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,328 @@
+
+
+.. _setup-systemd-unit:
+
+Start the server with systemd
+#############################
+
+Here is a systemd unit file that can be used to deploy the server on your machine.
+
+.. only:: oss
+
+    .. code-block:: 
+
+        [Unit]
+        Description=Podman 
+        Documentation=https://docs.inmanta.com
+        Wants=network-online.target
+        After=network-online.target
+        RequiresMountsFor=%t/containers
+
+        [Service]
+        Environment=PODMAN_SYSTEMD_UNIT=%n
+        Restart=on-failure
+        TimeoutStopSec=70
+        ExecStart=/usr/bin/podman run \
+                --cidfile=%t/%n.ctr-id \
+                --cgroups=no-conmon \
+                --sdnotify=conmon \
+                -d \
+                --replace \
+                --publish=127.0.0.1:8888:8888 \
+                --uidmap=993:0:1 \
+                --uidmap=0:1:993 \
+                --gidmap=993:0:1 \
+                --gidmap=0:1:993 \
+                --name=inmanta-orchestrator-server \
+                --volume=%E/inmanta/inmanta.cfg:/etc/inmanta/inmanta.cfg:z \
+                --entrypoint=/usr/bin/inmanta \
+                --user=993:993 \
+                ghcr.io/inmanta/orchestrator:latest \
+                -vvv --timed-logs server
+        ExecStop=/usr/bin/podman stop \
+                --ignore -t 10 \
+                --cidfile=%t/%n.ctr-id
+        ExecStopPost=/usr/bin/podman rm \
+                -f \
+                --ignore -t 10 \
+                --cidfile=%t/%n.ctr-id
+        Type=notify
+        NotifyAccess=all
+
+        [Install]
+        WantedBy=default.target
+
+.. only:: iso
+
+    .. code-block:: 
+       :substitutions:
+
+        [Unit]
+        Description=Podman 
+        Documentation=https://docs.inmanta.com
+        Wants=network-online.target
+        After=network-online.target
+        RequiresMountsFor=%t/containers
+
+        [Service]
+        Environment=PODMAN_SYSTEMD_UNIT=%n
+        Restart=on-failure
+        TimeoutStopSec=70
+        ExecStart=/usr/bin/podman run \
+                --cidfile=%t/%n.ctr-id \
+                --cgroups=no-conmon \
+                --sdnotify=conmon \
+                -d \
+                --replace \
+                --publish=127.0.0.1:8888:8888 \
+                --uidmap=993:0:1 \
+                --uidmap=0:1:993 \
+                --gidmap=993:0:1 \
+                --gidmap=0:1:993 \
+                --name=inmanta-orchestrator-server \
+                --volume=%E/inmanta/inmanta.cfg:/etc/inmanta/inmanta.cfg:z \
+                --volume=%E/inmanta/license/com.inmanta.license:/etc/inmanta/license/com.inmanta.license:z \
+                --volume=%E/inmanta/license/com.inmanta.jwe:/etc/inmanta/license/com.inmanta.jwe:z \
+                --entrypoint=/usr/bin/inmanta \
+                --user=993:993 \
+                containers.inmanta.com/containers/service-orchestrator:|version_major| \
+                -vvv --timed-logs server
```

Comment:
Like this the server logs will be sent to stdout and not to a file. Is this something we want? The RPM executes the following command to send the logs file: https://github.com/inmanta/inmanta-core/blob/ae61aa04b4b53447585a8ef055f5bb874dc6436e/misc/inmanta-server.service#L10

The same is true for the OSS service file of course.

Reply:
Yes, but that only works if the user didn't change the log folder configuration.  Or rather, it would always work, but be confusing if this log folder is not meant to be used.
I am not sure what is the best, I am open to both.

Reply:
As discussed we will go for the approach were:

* We set ` --log-file /var/log/inmanta/server.log`. That way the support archive also captures the server logs.
* And, we map the `/var/log/inmanta` directory by default to the hosts to make the logs accessible for the server in case the server fails to start.